flows/actions/testdata/send_broadcast.json

```diff
@@ -72,6 +72,26 @@
             "parent_refs": []
         }
     },
+    {
+        "description": "Error event if executed in batch start",
+        "as_batch": true,
+        "action": {
+            "type": "send_broadcast",
+            "uuid": "ad154980-7bf7-4ab8-8728-545fd6378912",
+            "urns": [
+                "tel:+1234567890"
+            ],
+            "text": "Hi there!"
+        },
+        "events": [
+            {
+                "type": "error",
+                "created_on": "2018-10-18T14:20:30.000123456Z",
+                "step_uuid": "59d74b86-3e2f-4a93-aece-b05d2fdcde0c",
+                "text": "can't send broadcasts during batch starts"
```

Comment:
is this maybe too strict?

Reply:
sending broadcasts? It is the original footgun so no, I don't think so.

Reply:
I thought maybe there's a valid case where your sending to one person.. like you start a bunch of people in a flow that messages their supervisor... so I was wondering if this should allow a broadcast to 1 URN or contact

Reply:
Hah, was literally just looking at that myself. But yes, should be consistent.